ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+     |             | Section 4.2.5 |                      | 2.3.12  |
+     +-------------+---------------+----------------------+---------+
+
+            Table 40: Properties of the VirtualLocation object
+
+4.  Updates to iCalendar
+
+4.1.  New Parameters
+
+4.1.1.  JSCALID
+
+   TBD
+
+4.2.  New Properties
+
+4.2.1.  JSCALID
+
+   TBD
+
+4.2.2.  JSPROP
+
+   TBD
+
+5.  Updates to JSCalendar
+
+5.1.  New Properties
+
+5.1.1.  completed
+
+   Name:
+      completed
+
+   Context:
+      Task
+
+   Type:
+      UTCDateTime (optional)
+
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 99]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   Description:
+      This is the date and time when the task was completed.
+
+5.1.2.  iCalComponent
+
+   Name:
+      iCalComponent
+
+   Context:
+      Any JSCalendar object
+
+   Type:
+      ICalComponent (optional)
+
+   Description
+      This contains information about an iCalendar component that got
+      partially or fully converted to JSCalendar.  It allows to preserve
+      the name of the iCalendar component and some or all of its
+      properties and subcomponents.
+
+      An ICalComponent object has the following properties:
+
+      @type: String (mandatory)  This specifies the type of this object.
+         This MUST be ICalComponent.
+
+      name: String (mandatory)  This is the name of the iCalendar
+         component in lowercase.
+
+      components: *[][] (optional)  This contains subcomponents of the
+         iCalendar component.  The value MUST be a list of iCalendar
+         components formatted in jCal as defined in Section 3.3 of
+         [RFC7265].
+
+      convertedProperties: String[ICalProperty] (optional)  This
+         contains conversion-related information about the component's
+         properties that got partially or fully converted to JSCalendar.
+         Each key defines the path to a property of the JSCalendar
+         object on which the iCalComponent property is set on.  The
+         value for each key contains information about the iCalendar
+         property which converted to the object property located at that
+         key (see Section 5.1.3).
+
+         The key MUST be a valid key of a PatchObject as defined in
+         Section 1.4.9 of [RFC8984].  The key MUST NOT point into a
+         nested property, unless there is no way to otherwise preserve
+         the iCalendar property elements.  For example, if a VEVENT
+         contains an ATTENDEE property, then any iCalendar information
+         about the ATTENDEE property MUST be set in the iCalProperty
+
+
+
+Stepanek                   Expires 16 May 2025                [Page 100]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+         property of the converted Participant object (see
+         Section 2.3.4).  In contrast, information about a RDATE
```

Comment:
I don't see any relevant examples in 2.3.4 or 2.3.35 for this large paragraph. It would be nice to have small inline examples here perhaps...